README.md

```diff
@@ -50,10 +50,9 @@ First, test that you can actually run `make`:
 > make
 formatted dhall files
 generated to packages.json
-validated packages' dependencies
 ```
 
-This is how you format Dhall files in the project, generate the `packages.json` that needs to be checked in, and validate that all dependencies declared in package definitions are at least valid. Unless you plan to consume only the `packages.dhall` file in your repository, you must check in `packages.json`.
+This is how you format Dhall files in the project and generate the `packages.json` that needs to be checked in. Unless you plan to consume only the `packages.dhall` file in your repository, you must check in `packages.json`.
```

Comment:
Why is validation not needed anymore?

Reply:
verify does it anyway

Reply:
validate was just something i used to speed up some from-bower workflows, but it's kind of useless other than for some quick local checking